tests/nat/cli/commands/mcp/test_mcp_cli.py

```
@@ -0,0 +1,761 @@
+
+    def schema_json(self, indent=2):
+        del indent
+        return json.dumps({"type": "object", "properties": {"a": {"type": "string"}}}, indent=2)
+
+
+def _install_fake_mcp(monkeypatch, *, list_tools_response=None, call_tool_result=None, ping_ok=True):
+    fake_mcp = ModuleType("mcp")
+
+    class _FakeClientSession:
+
+        def __init__(self, read, write):  # noqa: ARG002
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `ARG002`)

Remove unused `noqa` directive

(RUF100)